src/query/processor/cypher/astbuilder/ASTNode.cpp

```diff
@@ -0,0 +1,25 @@
+#include "ASTNode.h"
+#include <iostream>
+#include <string>
+#include <sstream>
+
+
+using namespace std;
+
+string ASTNode::print(int depth, string prefix, bool isLast) const
```

Comment:
May be good to keep this print function as a support for debugging. However, we should use a logger instead of outputting this to console.